.github/workflows/pr_test_peft_backend.yml

```diff
@@ -32,9 +32,7 @@ jobs:
           python -m pip install --upgrade pip
           pip install .[quality]
       - name: Check quality
-        run: |
-          ruff check examples tests src utils scripts
-          ruff format examples tests src utils scripts --check
+        run: make quality
```

Comment:
I think it makes more sense to just run the `make quality` command here instead of manually repeating the exact same commands we already have under the `quality` alias of our Makefile.